slate-video-library/src/Pages/Likespage/Likespage.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
+import { useLikeContext } from "../../Components/Context/LikespageContext";
+import { useSingleVideoContext } from "../../Components/Context/SingleVideopageContext";
+import Footer from "../../Components/Footer/Footer";
+import Header from "../../Components/Header/Header";
+import Sidebar from "../../Components/Sidebar/Sidebar";
+import SmallVideoCards from "../../Components/SmallVideoCards/SmallVideoCards";
```

Comment:
you can export all components in one file and then export all components from that file